components/contrib/kaggle-notebook-image/Dockerfile

```diff
@@ -41,9 +42,8 @@ RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
     curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - && \
     apt-get update && \
     apt-get install -y google-cloud-sdk kubectl && \
-    # pin to 0.8.1 due to conflicts with tornado>=5.x and pyzmq>=17.x
     # we just really need jupyterhub-singleuser for our KF scripts
-    pip install jupyterhub==0.8.1 \
+    pip install jupyterhub \
```

Comment:
Why not pin to a particular version of jupyterhub so it doesn't break unexpectedly?